js/skin.js

```diff
@@ -155,6 +160,7 @@ var Skin = React.createClass({
   /**
    * @public
    * @description set isVrMouseMove to false for click event
+   * @return {undefined}
```

Comment:
We should take this out since it doesn't return anything.

Reply:
\+